examples/grpc/src/test/java/example/armeria/grpc/GrpcReverseProxyServerTest.java

```diff
@@ -0,0 +1,63 @@
+package example.armeria.grpc;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.testcontainers.containers.BindMode;
+import org.testcontainers.containers.GenericContainer;
+import org.testcontainers.junit.jupiter.Container;
+
+import com.linecorp.armeria.client.WebClient;
+import com.linecorp.armeria.client.grpc.GrpcClients;
+import com.linecorp.armeria.server.Server;
+import com.linecorp.armeria.server.grpc.GrpcService;
+
+import example.armeria.grpc.Hello.HelloReply;
+import example.armeria.grpc.Hello.HelloRequest;
+
+class GrpcReverseProxyServerTest {
+
+    private static Server server;
+
+    @Container
+    private static GenericContainer<?> envoy = new GenericContainer<>("envoyproxy/envoy:v1.22.0")
+            .withExposedPorts(9090)
```

Comment:
Please use a random unused port to avoid port collision as much as possible. Use `PortUtil.unusedTcpPort`.